BrowserKit/Tests/TabDataStoreTests/TabDataStoreTests.swift

```diff
@@ -23,17 +23,15 @@ final class TabDataStoreTests: XCTestCase {
 
     // MARK: Saving Data
 
-    func testSaveTabData() async throws {
+    func testSaveTabData() {
```

Comment:
If you make the function async you can remove all of the Task blocks, same for the rest of the tests.